### DISCUSSION TITLE
Δυνατότητα ολοκλήρωσης του μαθήματος εξ' αποστάσεως

### DISCUSSION THREAD

**Discussion by VasilisG11** [courses-ionio/hci#2006] [Q&A] [answered]
Καλησπέρα κύριε @epidrome,

Ήθελα να ρωτήσω αν θα ήταν εφικτό να ολοκληρώσω το μάθημα σας εξ' αποστάσεως διότι μου είναι δύσκολο να είμαι τακτικά παρών για τις συναντήσεις. Ωστόσο έχω την ικανότητα να παρεβρεθώ για την μεσαία και τελική αναφορά προόδου.

**Comment by epidrome** [comment, marked as answer]
Ναι, φυσικά, το μάθημα είναι σχεδιασμένο για να τους εξυπηρετήσει όλους όπου και αν βρίσκονται, αρκεί να έρθεις δύο φορές για προφορική παρουσίαση προόδου, περίπου την 5η και 10η εβδομάδα

**Reply by VasilisG11** [reply]
Τέλεια, όσον αφορά τα βίντεο κουίζ. Επειδή μου είχαν κάνει καλή εντύπωση, θα υπάρχουν και φέτος στα παραδοτέα? Αν όχι θα σας ήταν εύκολο να τα μοιραστείτε?

**Reply by epidrome** [reply]
φέτος δεν έχει βίντεο κουίζ γιατί το ενδιαφέρον για το μάθημα είναι πολύ μικρό, οπότε η βαθμολογία του αντίστοιχου κριτηρίου γίνεται με τους άλλους τρόπους που περιγράφονται 
https://courses-ionio.github.io/help/classroom/
